PROJECT

```diff
@@ -1,42 +1,51 @@
 domain: openshift.io
-layout: go.kubebuilder.io/v2
+layout:
+- go.kubebuilder.io/v3
```

Comment:
AFAIK kubebuilder.io/v3 also requires other changes in our project directory & other places.

Reply:
We sorted this over a call.